src/github.com/matrix-org/dendrite/typingserver/dummy/api/api.go

```diff
@@ -0,0 +1,23 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import "github.com/matrix-org/gomatrixserverlib"
+
+// TODO: Remove this package after, typingserver/api is updated to contain a gomatrixserverlib.Event
```

Comment:
We should probably move this TODO out of dummy package now that the plan is no longer to change to using Event